### PR TITLE
Adding MagicalMoney pool

### PR DESCRIPTION
**Pull request by adamneilson**

### CODE REVIEW

**Review by leo42 — APPROVED** [review]

**Comment by leo42** [comment]
Hey, seems like you locked your branch. we need to resolve conflicts before merging :)

**Comment by adamneilson** [comment]
Hi @leo42 I've given you write access to `MagicalMoney:master` if that isn't what you are asking for, let me know. Seems the conflict is based around lots of simultaneous merges rather than anything wrong with the json itself so happy for you to fix it or do you require me to?

**Pull request merged**
